HELP.md

```diff
@@ -0,0 +1,9 @@
+# Getting Started
```

Comment:
I haven't seen a HELP file in any other Companie House projects. Is there a specfic reason it is included. If not I think we should stick with convention and put information in the RAEDME